Chaap/Chaap/Presentation/Main/Segments/CalendarSegmentView.swift

```diff
@@ -8,11 +8,237 @@
+            }
+        }
+        .padding(10)
+        .frame(
+            maxWidth: .infinity,
+            minHeight: 36,
+            maxHeight: 36,
+            alignment: .center
+        )
     }
+    
+    private var calendarGrid: some View {
+        VStack(alignment: .center, spacing: 5) {
+            ForEach(0..<6, id: \.self) { week in
+                HStack(alignment: .center, spacing: 10) {
+                    ForEach(0..<7, id: \.self) { day in
+                        let index = week * 7 + day
+                        if index < viewModel.daysInMonth.count {
+                            dayCell(date: viewModel.daysInMonth[index])
+                        } else {
+                            Spacer()
+                        }
+                    }
+                }
+                .padding(10)
+                .frame(
+                    maxWidth: .infinity,
+                    minHeight: 36,
+                    maxHeight: 36,
+                    alignment: .center
+                )
+            }
+        }
+    }
+    
+    private func dayCell(date: Date) -> some View {
+        let day = Calendar.current.component(.day, from: date)
+        let isSelected = Calendar.current.isDate(
+            date,
+            inSameDayAs: viewModel.selectedDate
+        )
+        let isToday = Calendar.current.isDateInToday(date)
+        let isCurrentMonth = Calendar.current.isDate(
+            date,
+            equalTo: viewModel.currentMonth,
+            toGranularity: .month
+        )
+
+        return Button {
+            viewModel.dateWasSelected(date)
+        } label: {
+            VStack(spacing: 0) {
+                ZStack {
+                    /// 선택 배경 원 (숫자 텍스트 중앙에 위치)
+                    Ellipse()
+                        .fill(viewModel.cellBackgroundColor(
+                            isSelected: isSelected,
+                            isToday: isToday
+                        ))
+                        .frame(width: 28, height: 27)
+
+                    Text("\(day)")
+                        .font(.chPrimaryCaptionRegular)
+                        .foregroundColor(viewModel.cellTextColor(
+                            for: date,
+                            isSelected: isSelected,
+                            isToday: isToday,
+                            isCurrentMonth: isCurrentMonth
+                        ))
+                }
+
+                /// 오늘 날짜 표시 (선택되지 않았을 때만)
+                if isToday && isCurrentMonth && !isSelected {
+                    Text("오늘")
+                        .font(.system(size: 9, weight: .bold))
+                        .foregroundColor(.chPrimary)
+                } else {
+                    /// 빈 공간 유지
+                    Text("")
+                        .font(.system(size: 9, weight: .medium))
+                        .frame(height: 10)
+                }
+            }
+            .frame(width: 35, height: 36)
+        }
+    }
+    
```

Comment:
```suggestion

    //TODO: Custom Modal로 변경해야 함.    
```

Hi-Fi에서는 해당 부분 뷰 내부의 리스트가 아닌 모달로 보여지기 때문에 추후 작업이 필요할 것 같습니다!